empirical-results.tex

```diff
@@ -1 +1,29 @@
-\section{Empirical Results}
\ No newline at end of file
+\section{Empirical Results}
+
+\newcommand{\rref}[1]{}
+
+In order to evaluate our consumption mechanisms, we implement the federated consumption mechanism in Solidity. We provide a concrete implementation of the \textsf{burn-verifier} contract described in \rref{alg.burn-verifier} as \textsf{BurnedBTC}. We implement the \textsf{Crosschain} parent contract from \cite{pow-sidechains}. In order to achieve this we implement our own verification methods similar to \rref{alg.verify-event-federation} in our \textsf{Verifier} library. Specifically, we verify transaction data in \textsf{Verifier} similarly to \rref{alg.verify-tx} by making use of the open-source bitcoin-spv library \cite{bitcoin-spv-library}. We verify block connection using a Merkle Mountain Range where the leaf corresponding to each block has the value of the block's transaction id merkle tree root.
+
+Finally, the \textsf{BurnVerifier} library provides a concrete implementation for our berning protocol. Specifically the library implements the \textsf{BurnVerify} function of the protocol.
+
+In our implementation, the federation is limited to one party for simplicity but it can be easily adapted for more.
+
+We release our implementation as open-source software under the MIT license
+\footnote{\url{https://github.com/decrypto-org/burn-paper/tree/master/experiment}}.
+The implementation is fully tested with 100\% code coverage.
```

Comment:
This paragraph can be moved to a footnote. Also let's anonymize this paper by introducing an `\ifanonymous` macro at this point to avoid name leaks in the FC submission.

Reply:
What would you like to be shown on the anon version?

Reply:
Yes. Say "The link to our open source software implementation, which uses the MIT license, has been redacted for anonymity. The implementation is production-ready and fully tested with 100\% code coverage."